bin/qunit-testrunner.js

```diff
@@ -152,7 +153,7 @@ const setupWebServer = async (options) => {
         middlewares
     });
 
-    return webServer.listen(port, host);
+    return webServer.listen(port, host, testDir);
```

Comment:
Why the webserver should know about `testDir`. It is totally irrelevant for it. If you would like to console.log it, then do it somewhere else. Even because if `--withoutServer` will be set, your line won't be on the console, but testDir still defined.

Reply:
@btamas thanks for the comment. Will think about it.

Reply:
Done.